build/Makefile.global

```diff
@@ -142,6 +142,12 @@ prof-clean:
 prof-use:
 	CCACHE_DISABLE=1 $(MAKE) PROF_FLAGS=-fprofile-use all
 
+%_arginfo.h: %.stub.php
+	php scripts/dev/gen_stub.php $<
```

Comment:
This might make building php from source a bit harder if file modification times are newer for the php file? (could add a check if php can be executed, e.g. `if type php >/dev/null 2>/dev/null; then... ; fi`)

Reply:
Done!